readme.md

```diff
@@ -12,3 +12,39 @@ libqmlbind is a C library for creating QML bindings for other languages easily t
   * High portability
 * Build with qmake
   * No need to configure compilation flags for Qt manually
+
+## How to build
+### Requirements
+* A C++11-capable compiler: libqmlbind is tested against `clang 3.4`.
+* Qt5: libqmlbind is tested against `Qt 5.5.1`. Everything `>= 5.2` might work, too.
+
+### Build the Code
+First, clone the repository with all its submodules:
+```
+> git clone --recursive git@github.com:seanchas116/libqmlbind.git
+```
+
+Alternatively, if you already cloned but forgot `--recursive`, use this inside `libqmlbind`:
+```
+> git submodule update --init --recursive
+```
+
+Use qmake to generate the makefile. The `-r` option is needed because `libqmlbind` depends on Qt's private headers.
```

Comment:
Makes sense! I was fooled there because as soon as I added the `-r` option as it was done in `.travis.yml`, the private header warnings

```
Project MESSAGE: This project is using private headers and will therefore be tied to this specific Qt module build version.
Project MESSAGE: Running this project against other versions of the Qt modules may crash at any arbitrary point.
Project MESSAGE: This is not a bug, but a result of using Qt internals. You have been warned!
```
showed up. :laughing: